src/redactor.rs

```diff
@@ -66,7 +66,7 @@ impl Redactor {
     /// # Example
     ///
     /// ```rust
-    /// # use veil::RedactorBuilder;
+    /// # use veil::redactor::RedactorBuilder;
```

Comment:
Would it be better to use `Redactor::builder` instead of `RedactorBuilder::new` here?

Reply:
Sure, that would be nicer